recipes/python-jsonpath/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "python-jsonpath" %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/python_jsonpath-{{ version }}.tar.gz
+  sha256: 5099d88e55d71314eac5d8b7acc528cb47d53783ba9685940da8fb63a7674b54
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  imports:
+    - python_jsonpath
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  summary: Another JSONPath implementation for Python.
+  dev_url: https://github.com/jg-rp/python-jsonpath
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - jacobtomlinson
```

Comment:
@jg-rp I see this package isn't on conda forge and I need it for my package. Are you happy with me adding this and are you ok with me adding you to this list?

Reply:
Yes, I'm happy. And I'm OK with being listed as a maintainer.

Reply:
Thank you!